lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +222,30 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = split_by_sentence(text)
+        conllu_sent = []
+        for id, sentence in enumerate(sentences, start=1):
+            conllu_sent.append(ConlluSentence(id, sentence, []))
```

Comment:
why do you leave token list empty?

Reply:
i changes the implementation, now  it is filled